src/main/java/org/chainoptim/desktop/features/client/controller/ClientOrdersController.java

```diff
@@ -59,9 +68,6 @@ public ClientOrdersController(FallbackManager fallbackManager,
     @Override
     public void setData(Client client) {
         this.client = client;
```

Comment:
You should load the FallbackManagerView here and place it in a container with parent StackPane like I did throughout the app. Also make sure to reset fallbackManager before starting to load orders.